liferay-echo/liferay-portal
# LPD-19160 Allow move Display page templates and multiple items at the same time

> [!NOTE]
>You need the FF  LPS-189856.

## Motivation

[Link to ticket](https://liferay.atlassian.net/browse/LPD-19160)

### Changes

Add the bulk action to move folders and display page templates.

## Test scenario 

1. Create a DPT
2. Create some folders 
3. Check the move action for each one individually using the dropdown
4. Check the move action in bulk slecting different elements
